CRM/Admin/Form/Preferences/Display.php

```diff
@@ -137,6 +136,12 @@ public function buildQuickForm() {
     $this->addElement('select', 'editor_id', ts('WYSIWYG Editor'), $wysiwyg_options, $extra);
     $this->addElement('submit', 'ckeditor_config', ts('Configure CKEditor'));
 
+    $this->addRadio('contact_ajax_check_similar', ts('Check for Similar Contacts'), array(
+      '1' => ts('While Typing'),
+      '0' => ts('When Saving'),
+      '2' => ts('Never'),
```

Comment:
Note: In all existing installs this setting is already set to either '1' or '0'. These new options keep the original meaning of 1 and 0 so no upgrade script is needed.